### ISSUE TITLE
AttributeError: 'module' object has no attribute '_CallList'

### ISSUE THREAD

**Issue by danielgatis** [gabrielfalcao/sure#80]
Hi,
When I run my tests I get the same error for all tests:

```
  File "/home/vagrant/.tox/project/py34/lib/python3.4/site-packages/sure/__init__.py", line 377, in wrapper
    value = func(self, *args, **kw)
  File "/home/vagrant/.tox/project/py34/lib/python3.4/site-packages/sure/__init__.py", line 600, in equal
    comparison = DeepComparison(self.obj, what, epsilon).compare()
  File "/home/vagrant/.tox/project/py34/lib/python3.4/site-packages/sure/core.py", line 241, in compare
    if isinstance(X, mock._CallList):
AttributeError: 'module' object has no attribute '_CallList'
```

This is strange because the same test worked, and I dont know what I did to them breaking.
How can I fix this?

**Comment by danielgatis** [comment]
fix here: https://github.com/gabrielfalcao/sure/pull/81

**Comment by gabrielfalcao** [comment]
closed at #81

**Comment by gabrielfalcao** [comment]
1.2.23 is out
